src/leap/mail/imap/service/imap.py

```diff
@@ -146,16 +208,8 @@ def run_service(store, **kwargs):
     :returns: the port as returned by the reactor when starts listening, and
```

Comment:
documented in both places. my idea is to bring here the mail multi-services (they are now in bitmask_core.mail_services), so those would be the main entrypoints. run_service would be a "private" method then.